CoffeeShop/src/main/java/workers/Chef.java

```diff
@@ -0,0 +1,179 @@
+package workers;
+
+import exceptions.InvalidItemIDException;
+import interfaces.Observer;
+import item.ItemList;
+import logs.CoffeeShopLogger;
+import order.FoodList;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Class represents how a Chef functions in the Coffee Shop Simulation
+ *
+ * This class uses two design patterns:
+ * 1. Factory Design Pattern (with Staff and StaffFactory)
+ * 2. Observer Design Pattern (between this class and FoodList)
+ *
+ * Tasks:
+ * 1. Check for Item by checking for items in FoodList
+ * 2. Wait specified amount of time to complete Item
+ * 3. Add Item to waiters list for processing
+ *
+ * @author Fraser Holman
+ */
+public class Chef extends Staff<String> {
+    FoodList foodList;
+
+    ItemList itemList;
+
+    Map.Entry<Waiter, String> currentItem;
+
+    StaffList staffList;
```

Comment:
guessing we want private keywords here?
```suggestion
    private FoodList foodList;

    private ItemList itemList;

    private Map.Entry<Waiter, String> currentItem;

    private StaffList staffList;
```